build/rbac/rbac.yaml

```diff
@@ -953,18 +921,6 @@ rules:
       - create
       - update
       - delete
-  - apiGroups:
```

Comment:
There is some overlap of this with the monitoring/rbac.yaml. What if we keep this, add it to common.yaml, and remove the overlap in the monitoring/rbac.yaml. If we remove this from the helm chart, it's possible some clusters will be broken that were using monitoring.

Reply:
I will undo this change and leave this in for now, but I'm not sure it's good to add into `common.yaml`. After all, monitoring is an optional thing, and the RBAC should be as well. 

Monitoring is currently part of the new cluster helm chart in `cluster/charts/rook-ceph-cluster/templates/role{binding}.yaml` with the optional value `monitoring.enable`. I can do a future update that makes the operator chart able to deploy the monitoring rbac as well (optionally), and if I enable monitoring by default, then we shouldn't break any existing helm users who are also using monitoring. Then I can also remove this section since it won't be needed.